documentation/release_notes.markdown

```diff
@@ -61,33 +91,56 @@ This release retires all api versions before v4 and promotes v4 to be
+
+* We've retired some previously deprecated commands ([PDB-840](https://tickets.puppetlabs.com/browse/PDB-840))
 
     The currently supported commands and versions are now:
 
     * replace catalog, v6
     * replace facts, v4
-    * deactivate node, v2
+    * deactivate node, v3
     * store report, v5
 
-* All API commands and responses now use underscores, instead of dashes.  ([PDB-698](https://tickets.puppetlabs.com/browse/PDB-698))
+    We've insured that internally the previous version of the commands will still be processed, so you
+    don't lose the data being processed in your queue during an upgrade.
+
+* All API commands and responses now use underscores, instead of dashes. ([PDB-698](https://tickets.puppetlabs.com/browse/PDB-698))
 
     This patch changes our API commands and responses to use underscores instead of
     dashes, which will make it easier to interface with PDB using languages like
-    javascript and python.
+    javascript and python. Consult the various API endpoints for the exact new
+    parameters if you are uncertain.
+
+* Split the comman endpoint (`/v3/commands` or `/v4/commands`) into their own
+  endpoint namespace served at `/pdb/cmd/v1`. ([PDB-1534](https://tickets.puppetlabs.com/browse/PDB-1534)) ([PDB-1561](https://tickets.puppetlabs.com/browse/PDB-1561))
+
+    The commands endpoint has now been separated from queries so that we can
+    version and manage this endpoint separately.
+
+* The `/metrics` endpoint now has its own top level namespace `/pdb/metrics/v1` ([PDB-800](https://tickets.puppetlabs.com/browse/PDB-800))
+
+    This is something we'd been planning on doing for a while, but separation
+    means we can manage and version this endpoint separately. There are plans
+    to have this endpoint supported in other Puppet Labs products also, and
+    this move gears us up for that.
+
+* Version and server-time now belongs in its own endpoint (meta) ([PDB-1562](https://tickets.puppetlabs.com/browse/PDB-1562))
 
-* Split the `/v3/commands` endpoint into its own API served at `/pdb/cmd/v1`.
+    These endpoints now live in `/pdb/meta/v1`.
 
-* Change "name" key to "certname" ([PDB-1099](https://tickets.puppetlabs.com/browse/PDB-1099))
+* Change `name` key to `certname` across our API ([PDB-1099](https://tickets.puppetlabs.com/browse/PDB-1099))
 
-    For clarity and consistency across endpoints, the "name" key has
-    been changed to "certname" where appropriate. This affects
-    responses from the /pdb/query/v4/catalogs endpoint, queries to it, and the
-    "replace catalog" and "replace facts" commands.
+    For clarity and consistency across endpoints, the `name` key has
+    been changed to `certname` where appropriate. This affects
+    responses from the `/pdb/query/v4/catalogs` endpoint, queries to it, and the
+    `replace catalog` and `replace facts` commands.
 
 * Expanded data format is now more formalized. ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-1228))
```

Comment:
we should figure out a way to express this more clearly. "Expanded data format" doesn't mean anything to the consumers of this document, and didn't exist prior to this upgrade.